src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -223,6 +223,7 @@ public function has($id)
             if ('service_container' === $id
                 || isset($this->aliases[$id])
                 || isset($this->services[$id])
```

Comment:
IUUC, `$this->services` always exist and is always an array, so this line can be removed (the `array_key_exists` call in the next will do the check).